docs/en/trust.rst

```diff
@@ -2,56 +2,298 @@
+
+.. code-block:: python
+
+    {
+        "alg": "RS256",
+        "kid": "em3cmnZgHIYFsQ090N6B3Op7LAAqj8rghMhxGmJstqg",
+        "typ": "entity-statement+jwt"
+    }
+    .
+    {
+        "exp": 1649623546,
+        "iat": 1649450746,
+        "iss": "https://intermediate.eidas.example.org",
+        "sub": "https://rp.example.it",
+        "jwks": {
+            "keys": [
+                {
+                    "kty": "EC",
+                    "kid": "2HnoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs",
+                    "crv": "P-256",
+                    "x": "1kNR9Ar3MzMokYTY8BRvRIue85NIXrYX4XD3K4JW7vI",
+                    "y": "slT14644zbYXYF-xmw7aPdlbMuw3T1URwI4nafMtKrY",
+                    "x5c": [ <X.509 certificate> ]
+                }
+            ]
+        },
+        "metadata_policy": {
+            "openid_relying_party": {
+                "scope": {
+                    "subset_of": [
+                         "eu.europa.ec.eudiw.pid.1",  
+                         "eu.europa.ec.eudiw.pid.1:given_name",
+                         "email"
+                      ]
+                },
+               "request_authentication_methods_supported": {
+                "one_of": ["request_object"]
+                },
+               "request_authentication_signing_alg_values_supported": {
+                    "subset_of": ["RS256", "RS512", "ES256", "ES512", "PS256", "PS512"]
+               }
+            },
+            "client_metadata": {
+                "vp_formats": {
+                    "jwt_vp": {
+                        "alg": 
+                            "subset_of": [
+                                "EdDSA",
+                                "ES256K"
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+
+
+Trust Evaluation Mechanism
+--------------------------
+
+The Trust Anchor publishes the list of its Intermediates (Federation Subordinate Listing endpoint) and the attestations of their metadata and public keys (Entity Statements). 
+
+Each participant, including Trust Anchor, Intermediate, Credential Issuer, Wallet Provider, and Relying Party, publishes its own metadata and public keys (Entity Configuration endpoint) on the well-known web resource **.well-known/openid-federation**.
+
+Each of these can be verified using the Entity Statement issued by a superior, Trust Anchor, or Intermediate.
+
+Each Entity Statement is verifiable over time and has an expiration date. The revocation of each statement is verifiable in real time and online (only for remote flows) through the federation endpoints.
+
+The concatenation of the statements, through the combination of these signing mechanisms and the binding of claims and public keys, creates the Trust Chain. This contains Trust Marks as well as protocol-specific metadata, where Trust Anchor metadata policies are applied.
```

Comment:
We should also define the role of the Trust Marks in this EUDIW context. How are they used? How are they defined? Which are the main trust marks profile we consider?

Reply:
Before defining Trust Marks I need to have the use cases where Trust Marks are needed, since we don't have anymore the problem of the federation entity discovery abuse